web/source/keymanweb.js

```diff
@@ -4098,7 +4098,7 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
       opt['fonts'] = fixPath(opt['fonts']);    
 
       // Set element attachment type    
-      if(opt['attachType'] == '') opt['attachType'] = (device.touchable ? 'manual' : 'auto');
+      if(opt['attachType'] == '') opt['attachType'] = 'auto';
 
   /*    
```

Comment:
I notice that the code you're changing here is repeated again (without your change) in the large block of commented out code below. Do you know why this code is commented out (rather than deleted)? If there is a reason to keep it around, a comment would be nice to explain that, and if it is likely to be coming back, should your change be made in the commented-out version as well?

Reply:
Honestly, it's probably prime for deletion, and I'm not sure why it's still there.  @mcdurdin, thoughts?

I figure its removal would probably be best in its own PR or one oriented toward reformatting/cleaning the codebase.

Reply:
I agree that if it's going away, a separate PR makes sense.